views/login.hbs

```diff
@@ -24,4 +21,63 @@
 		<a href="/organisation/new" class="d-block d-lg-inline ms-lg-5">I want to register my band</a>
 	</form>
 
-</div>
\ No newline at end of file
+</div>
+
+<script>
+	const error = document.getElementById("error");
+	const form = document.getElementById("login");
+	const email = document.getElementById("email");
+	const password = document.getElementById("password");
+
+	function showErrors(errors){
+		error.innerHTML = errors.map(e => e.message).join("<br />");
+		error.classList.remove("d-none");
+	}
+
+	form.addEventListener("submit", async (e) => {
+		e.preventDefault();
+		error.classList.add("d-none");
+		email.disabled = true;
+		password.disabled = true;
+		
+		let response = await fetch("https://dev.payload.byba.online/api/users/login", {
+			method: 'POST',
+			headers: {
+				"Content-Type": "application/json"
+			},
+			credentials: 'include',
+			body: JSON.stringify({
+				email: email.value,
+				password: password.value
+			})
+		});
+
+		let json = await response.json();
+
+		if(response.status / 100 != 2){
```

Comment:
this is just the same as `=200` right?

Reply:
To be fair yeah lmao. I originally had `== 4` so it would catch any errors but I suppose the divisions aren't needed now!